ECOSYSTEM.md

```diff
@@ -4,24 +4,33 @@ We aim to build a vibrant and inclusive ecosystem of partners, core contributors
 
 There are three types of stakeholders:
 
-* **Partners:** Companies that are significantly invested in React Native and have been for years.
+* **Partners:** Companies significantly invested in React Native and take responsibility for the React Native vision and community.
 * **Core Contributors:** Individual people who contribute to the React Native project.
 * **Community Contributors:** Individuals who support projects in the [react-native-community](https://github.com/react-native-community) organization.
 
 ## Partners
 
-Partners are companies that are significantly invested in React Native and have been for years. Informed by their use of React Native, they push for improvements of the core and/or the ecosystem around it. Partners think of React Native as a product: they understand the trade offs that the project makes as well as future plans and goals. Together we shape the vision for React Native to make it the best way to build applications.
+Partners are companies that are significantly invested in React Native and demonstrate ownership. Informed by their use of React Native, they push for improvements of the core and/or the ecosystem around it. Examples of this may include large scale contributions to `react-native` or owning an essential tools or libraries. 
 
-React Native's current set of partners include Callstack, Expo, Infinite Red, Meta, Microsoft and Software Mansion. Many engineers from these companies are core contributors, and their partner responsibilities also include:
+Partners think of React Native as a product: they understand the trade offs that the project makes as well as future plans and goals. Together we shape the vision for React Native to make it the best way to build applications. 
 
-* **[Callstack](https://callstack.com/):** Manages releases, maintains the [React Native CLI](https://github.com/react-native-community/react-native-cli) and organizes [React Native EU](https://react-native.eu/)
+Baseline responsibilities of partners include: 
```

Comment:
other potential options, just throwing there for brainstorming:
* help maintain the website and showcase (example of this right now: Simek from Expo)
* produce content/material/blogposts to showcase their usage of react-native on a consistent basis

Reply:
Great suggestions! One thing to clarify -- I was under the impression we really need more community commitments for release co-pilot. My concern is that if we make the baseline responsibilities too broad, then we might end up here again. The responsibilities of partners should be meeting baseline and in-addition demonstrating ownership. 

I think the points you listed fit well with demonstrating ownership-- we should also update Expo's list to include the website maintenance (cc @Simek if that's right by you)

Reply:
@Simek has been contributing to the website off-the-clock, but we'll discuss whether we want to possibly make this an official responsibility. i won't have an answer soon and we shouldn't hold up this doc on it

Reply:
yeah my point is basically related to what @brentvatne is saying in his other comment about ways of being involved:

> expo can certainly help out as a community releaser/release co-pilot in the future, although i wonder if it is useful to require that every partner contribute in this particular way. for example, our role in recent releases has been to provide our feedback from integrating the new release into our tooling and also give technical release support in the form of pull requests and reviews for pull requests. @Kudo has been handling this.

But yes, we'd like to get a couple more people that can be part of the release crew from the community perspective. I'm just concerned that some companies might shy away from becoming a partner if we set that as an "hard requirement"

Reply:
I see, I defer then to you guys what you think is necessary to state as baseline requirements -- or if there should be any at all. My impression is that it is difficult to get consistent commitment from the community for release support and leveraging our partners (which of course they're already doing) could help. 

We could make the baseline requirement still "support releases" but in a more broad definition as @brentvatne shared or get rid of it.

Reply:
for this pass it may be appropriate to keep the definition broad, and we can refine that as we gain a better understanding of where different partners can contribute, and what we think should be a fair contribution level to balance the upsides of being a partner organization